docs/src/content/docs/sdk.mdx

```diff
@@ -37,21 +75,73 @@ The session status is based on the `UltravoxSessionStatus` enum and can be one o
 | thinking      | The model is processing/thinking.           |
 | speaking      | The model is speaking.                      |
 
+#### Status Events
 The status can be retrieved by adding an event listener to the state:
 
-```javascript
-import { UltravoxSession } from 'ultravox-client';
+<Tabs syncKey="example-language">
+  <TabItem label="JavaScript">
+    ```js
+    import { UltravoxSession } from 'ultravox-client';
```

Comment:
This will be easier to maintain if we have less repeated code.  Can we say something like "with `state` as defined above" and then remove everything that's not directly related to listening?

i.e. just
```
state.addEventListener('ultravoxSessionStatusChanged', (event) => {
  console.log('Session status changed: ', event.state);
});
```
for JS

Reply:
yeah good suggestion.